File: src/main/java/com/sonndc/dto/RoomBookingDTO.java
package com.sonndc.dto;

import lombok.Data;

@Data
public class RoomBookingDTO {
	private long id;
	private long roomId;
	private long userId;
	private String date;
	private String time;
	private String comment;
	
	public RoomBookingDTO(long roomId, long userId, String date, String time, String comment) {
		super();
		this.roomId = roomId;
		this.userId = userId;
		this.date = date;
		this.time = time;
		this.comment = comment;
	}
}

File: src/webapp/frontend/src/models/RoomBookingDto.ts
/* eslint-disable */
export class RoomBookingDto {
  public id?: string;
  public roomId?: string;
  public userId?: string;
  public date?: string;
  public time?: string;
  public comment?: string;

  constructor(init?: Partial<RoomBookingDto>) {
    Object.assign(this, init);
  }
}
